src/lib/swapper/swappers/ThorchainSwapper/utils/getL1ToLongtailQuote.ts

```diff
@@ -37,7 +37,18 @@ export const getL1ToLongtailQuote = async (
     slippageTolerancePercentageDecimal,
   } = input
 
-  const { chainNamespace } = fromAssetId(sellAsset.assetId)
+  const longtailTokensJson = await import('../generated/generatedThorLongtailTokens.json')
+  const longtailTokens: AssetId[] = longtailTokensJson.default
+
+  if (!longtailTokens.includes(buyAsset.assetId)) {
+    return Err(
+      makeSwapErrorRight({
+        message: `[getThorTradeQuote] - Unsupported assetId ${buyAsset.assetId}.`,
+        code: TradeQuoteError.UnsupportedTradePair,
+        details: { buyAssetChainId: buyAsset.chainId },
```

Comment:
Probably want to include the full `buyAsset` and `sellAsset` here, or at least `buyAssetId` and `sellAssetId`

Reply:
Added both full objects